eskip/lexer.go

```diff
@@ -291,6 +297,7 @@ func selectVaryingScanner(code string) scanner {
 	var sf scannerFunc
 	switch code[0] {
 	case '/':
+	case '#':
```

Comment:
this would be really `case '/', '#':` in go, while this way nothing happens in `case '/':`

Reply:
Oh wow, I guess I'm too used to PHP where it will fallthrough.